pack/tdc/tdcp.json

```diff
@@ -0,0 +1,515 @@
+        "skill_intellect": 2,
+        "skill_willpower": 2,
+        "subname": "The Forgotten Child",
+        "text": "Cannot be chosen as a starting investigator.\nYou cannot add, remove, or upgrade cards from your deck.\nYou get +2 skill value while resolving abilities on [[Spell]] cards.\n<b>Forced</b> - When you are defeated: You are <b>killed</b>.\n[elder_sign] effect: +0. Draw 1 card.",
+        "traits": "Construct. Scion.",
+        "type_code": "investigator"
+    },
+    {
+        "code": "11072",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Ethan Patrick Harris",
+        "name": "Whispers of Doom",
+        "pack_code": "tdcp",
+        "position": 72,
+        "quantity": 2,
+        "skill_combat": 1,
+        "skill_willpower": 1,
+        "text": "<b>Parley.</b> Choose a non-weakness enemy at your location and test [willpower] (3). This test gets +X difficulty, where X is that enemy's horror value. If you succeed, defeat that enemy (or deal it 3 damage instead if it is [[Elite]]). If you fail, place 1 doom on that enemy.",
+        "traits": "Spell. Cursed.",
+        "type_code": "event",
+        "xp": 0
+    },
+    {
+        "code": "11083",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "flavor": "\"Each block of clay hides all manner of fantastical things.\"",
+        "health": 1,
+        "illustrator": "Romana Kendelic",
+        "is_unique": true,
+        "name": "Lawrence Carlisle",
+        "pack_code": "tdcp",
+        "position": 83,
+        "quantity": 2,
+        "sanity": 1,
+        "skill_intellect": 1,
+        "slot": "Ally",
+        "subname": "Sculpting His Dreams",
+        "text": "You get +1 [intellect].\n[reaction] After you discard 1 or more cards from your hand during your turn, exhaust Lawrence Carlisle: Gain 1 resource.",
+        "traits": "Ally. Artist. Patron.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "11092",
+        "cost": 1,
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "illustrator": "Borja Pindado",
+        "name": "Improvised Weapon",
+        "pack_code": "tdcp",
+        "position": 92,
+        "quantity": 2,
+        "text": "You may play Improvised Weapon from your discard pile. If you do, shuffle it into your deck after resolving its effects and draw 1 card.\n<b>Fight.</b> The attacked enemy gets -2 fight for this attack. (If you played Improvised Weapon from your discard pile, this attack deals +1 damage.)",
+        "traits": "Tactic. Improvised.",
+        "type_code": "event",
+        "xp": 2
+    },
+    {
+        "pack_code": "tdcp",
+        "code": "11099",
+        "position": 99,
+        "cost": 2,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Pixoloid Studios",
+        "is_unique": true,
+        "name": "Library Pass",
+        "quantity": 1,
```

Comment:
Could be quantity 2, I guess?

Reply:
I'll leave this open until we have confirmation